docs/releases.rst

```diff
@@ -0,0 +1,51 @@
+.. _releasing:
```

Comment:
Not married to anything written here but I wanted to separate the documentation on what the deprecation policy is and the documentation of using the deprecation helpers.

Reply:
I like it :+1: